spec/features/merge_pact_spec.rb

```diff
@@ -0,0 +1,57 @@
+describe "Merging a pact" do
+
+  let(:pact_content) { load_fixture('a_consumer-a_provider.json') }
+  let(:path) { "/pacts/provider/A%20Provider/consumer/A%20Consumer/version/1.2.3" }
+  let(:response_body_json) { JSON.parse(subject.body) }
+
+  subject do
+    patch path, pact_content, {'CONTENT_TYPE' => 'application/json' }; last_response
+  end
+
+  context "when a pact for this consumer version does not exist" do
+    it "returns a 201 Created" do
+      expect(subject.status).to be 201
+    end
+
```

Comment:
I'm not keen on having PATCH create the resource when it doesn't exist, but I can see why you did it.